stories/index.js

```diff
@@ -666,6 +664,69 @@ storiesOf('TableHeading', module)
 
 storiesOf('Table', module)
   .add('base table', () => {
+    const noResults = props => (
+      <p>Nothing!</p>
+    );
+
+    return (
+      <Table
+        NoResults={noResults}
+      />
+    );
+  })
+
+  .add('empty with columns', () => {
+    const components = {
+      Table: ({ TableHeading, TableBody, NoResults, style, visibleRows }) => (
+        <table style={style}>
+          <TableHeading />
+          { visibleRows ? (TableBody && <TableBody />) : (NoResults && <NoResults />) }
+        </table>
+      ),
+      NoResultsContainer: compose(
```

Comment:
This seems to work as expected... Is there a reason your `Container` components use the following pattern?

```javascript
const WhateverContainer = OriginalComponent => compose(
  ...
)(props => <OriginalComponent {...props} />);
```